mlflow/deployments/interface.py

```diff
@@ -0,0 +1,55 @@
+from mlflow.deployments.plugin_manager import DeploymentPlugins
+from mlflow.utils import experimental
+
+
+listType = list
+plugin_store = DeploymentPlugins(auto_register=True)
+
+
+@experimental
+def get_deploy_client(target_uri):
+    """
+    It calls the `get_deploy_client` method from the plugin and return an instance of a subclass of
+    :py:class:`mlflow.deployments.BaseDeploymentClient` that can be used to deploy models to the
+    specified target
+
+    :param: target_uri: URI of target to deploy to. Run ``mlflow deployments --help`` via the CLI
+                        for more information on supported deployment targets
+    """
+    # TODO: Maybe we should keep the separator as ":/" instead of "://"?
```

Comment:
Agreed, `:/` seems like the right syntax for a URI, thanks for catching this

Reply:
Oops, ^I was wrong, `://`  is actually the right syntax: https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Definition

Reply:
Alright, let's go with `://` (although we have `:/` for models and runs)